server/db/user.js

```diff
@@ -9,40 +9,46 @@ const User = db.define("user", {
     defaultValue: UUIDV4
   },
   firstName: {
-    type: STRING,
-    allowNull: false,
-    validate: {
-      notEmpty: true
-    }
+    type: STRING
+    // allowNull: false,
```

Comment:
Why is the validating on this commented out?

Reply:
Just needed to turn off validations to get the oAuth up and running. I'll have to create a default user to pass to oAuth to pass through the User model validation

Reply:
You can uncomment it but don't use oAuth login yet